src/api/app/controllers/staging_project/staged_requests_controller.rb

```diff
@@ -1,5 +1,61 @@
 class StagingProject::StagedRequestsController < StagingProjectController
+  before_action :set_staging_workflow, :set_project, :set_xml_hash
+
   def index
-    @requests = @project.staged_requests
+    @requests = @staging_project.staged_requests
+  end
+
+  # accepts a list of bs_request numbers e.g.
+  # tosc api -X POST --data "<requests><number>1</number><number>2</number></requests>" '/staging_project/home:Admin:Staging:A/staged_requests'
```

Comment:
I think we can remove this comment now also, that was just a helper for me while testing...